web/src/views/HomeView/HomeView.vue

```diff
@@ -23,15 +23,11 @@
           >
             <v-col class="splash-text my-12">
               <div class="text-h2 font-weight-thin text-center light-blue--text text--darken-1">
-                LINC Archive
+                LINC Data Platform
               </div>
               <br>
               <div class="text-h6 font-weight-light text-center">
-                A cloud infrastructure and data archival platform for <a target="_blank" href="https://connects.mgh.harvard.edu/">LINC</a>, helping develop novel technologies for imaging brain connections down to the microscopic scale
-              </div>
-              <br>
-              <div class="text-h6 font-weight-light text-center">
-                This website is a fork of the DANDI Archive project. For more information, please visit <a target="_blank" href="https://dandiarchive.org/">the DANDI Archive</a>
+                A private data sharing and visualization platform for the <a target="_blank" href="https://connects.mgh.harvard.edu/">LINC Project</a>, helping develop novel technologies for imaging brain connections down to the microscopic scale.
```

Comment:
Just double-checking -- is there something more specific than `LINC Project` (e.g. reference to CONNECTS, etc). or is this sufficient?

Reply:
Thanks @aaronkanzer.  I rephrased the sentence since I wanted to move away from using the term `archive` since it has the connotation of being static.  And I decided not to add the NIH BRAIN Initiative CONNECTS Program to keep this description brief since this site is not the project homepage, but we do acknowledge the CONNECTS Program in the footer.    I am sure that this description will change over the years as we better understand the project and the broader landscape.